src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -237,7 +243,7 @@ static EffectBlureInit2 sBlureInit = {
     0, 2, 0, { 0, 0, 0, 0 },         { 0, 0, 0, 0 },
 };
 
-static s32 D_8092C198 = 0;
+static s32 texturesDesegmented = 0;
```

Comment:
Also, these are displaylists, not textures. If you look at what they are you'll probably get a better idea of what to call this, since it's a very particular set of displaylists.

Reply:
The display lists are not visible from what I can see, my best guess is it has to do with the armor coloring/effects, but not sure how to figure it out definitively.

Reply:
Yeah, for these you are going to have to look the at macro data itself in `object_ik.c`. Looks like they all just set different colors.